### PR TITLE
Update to DisplayLink v5.6.0 Offical Release

### PR DESCRIPTION
**Pull request by GrzegorzBB**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # Versions
 #
 
-DAEMON_VERSION := 5.5.0-59.151
+DAEMON_VERSION := 5.6.0-59.176
 DOWNLOAD_ID    := 4391    # This id number comes off the link on the displaylink website
-VERSION        := 1.10.1
-RELEASE        := 1
+VERSION        := 1.11.0
+RELEASE        := 2
```
Comment (elguero) [inline Makefile:8]: This should be '1' since this is the first release with this driver version.

File: displaylink.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
-%{!?_daemon_version:%global _daemon_version 5.5.0-59.151}
-%{!?_version:%global _version 1.10.1}
-%{!?_release:%global _release 1}
+%{!?_daemon_version:%global _daemon_version 5.6.0-59.176
+%{!?_version:%global _version 1.11.0}
+%{!?_release:%global _release 2}
```
Comment (elguero) [inline displaylink.spec:3]: Release '1'

**Review by elguero — CHANGES_REQUESTED** [review]
Thank you for the contribution.

**Comment by GrzegorzBB** [comment]
Thanks for the tips!

**Review by GrzegorzBB — COMMENTED** [review]
Version changed, I hope now it is everything.

**Comment by ParadoxGuitarist** [comment]
Just wanted to say I built this on Fedora 36 after modifying the Makefile to remove the i386 stuff and seems like it's working well!

**Comment by dominikzogg** [comment]
did it as @ParadoxGuitarist explained, works

**Comment by elguero** [comment]
Thank you for your contribution!

**Pull request merged**